portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,15 +57,33 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Helper function that returns an Array of strings with all the comments in Datastore
+   * Helper function to get and attempt to parse an integer parameter in request's query string.
```

Comment:
I think null and -1 both work fine here as long as it is documented in the comment =)